### ISSUE TITLE
Set the `GH_TOKEN` environment variable in `.travis.yml`

### ISSUE THREAD

**Issue by ExE-Boss** [viperHTML/viperhtml.github.io#23]
Now that #22 has been merged, step 5 can be completed:
> 5. Make a commit replacing the `GH_TOKEN` environment variable with an actual GitHub deploy token following [this guide](https://docs.travis-ci.com/user/environment-variables/#Defining-encrypted-variables-in-.travis.yml): https://github.com/viperHTML/viperhtml.github.io/blob/bfe1a7166474701d49064ac778b47110f8ab1685/.travis.yml#L4 and push the commit to the `src` branch (make sure to use the `api.travis-ci.com` API endpoint to get the encryption keys and not the legacy `api.travis-ci.org` API endpoint, otherwise Travis will fail to decrypt the `GH_TOKEN`).

**Comment by WebReflection** [comment]
about the branch thing, this repo is configured to use master for the web, no gh-pages

**Comment by WebReflection** [comment]
but I am going to sleep and travis is not working as expected so no rush at all

**Comment by WebReflection** [comment]
so ... I've no idea how to move forward in here. There is no panel saying add deploy token anywhere in here.

**Comment by ExE-Boss** [comment]
You need to install the [Travis command line application](https://github.com/travis-ci/travis.rb#installation) and run <code>travis encrypt 'GH_TOKEN=[{replace this with an actual GitHub token with the repo permission}](https://developer.github.com/apps/building-oauth-apps/understanding-scopes-for-oauth-apps/#available-scopes)'</code> and then paste the result into `.travis.yml`, overwriting line 4:
https://github.com/viperHTML/viperhtml.github.io/blob/7b85044a045782dcc9e7873ab1b48f3c01b9ab74/.travis.yml#L4

**Comment by WebReflection** [comment]
> and run travis encrypt 'GH_TOKEN={replace this with an actual GitHub token with the repo permission}'

I've actually run `travis encrypt {the actual GitHub token}` instead, without including `GH_TOKEN=` which I assumed was wrongly there ... is that correct?

If so, can you please verify all is good now? Thanks.

**Comment by WebReflection** [comment]
right ... something tells me it didn't work ...

```
[0K$ ./bin/build ../gh-pages

if [ $# -ge 1 ]; then
  dir=$1
  if [ ! -d "$basedir/$dir" ] || [ ! "$(ls -A "$basedir/$dir")" ]; then
    git clone -b master -v https://${GH_TOKEN}@github.com/viperHTML/viperhtml.github.io.git "$basedir/$dir"
  fi
  rm -rf "$basedir/$dir/*"
fi
Cloning into './bin/../../gh-pages'...
fatal: unable to access 'https://EOha0C3HdkKFv97pvBX8QZ7tUXu9QeTamoPuocpHjVTOZxychftqr3EsW9PaSIDjaFSPyKUUywCsf1w53q5NGP11EZpCvEq5bjISDAy5mQnPtB0+J7orVy8lQH5npsYWXSBMqOL6uBOiTWTsgWRVDyni6REeCy8ECEwqBsDIEfZp/1AE2YuWD7ASlUCOOsyWaJbnDuToYgZsm46YjCJoCGpmXR0xBtZvMSag2ruHXApoTPIP6KNfDXg/aVHgpoa+kEw89aiAg8+zHnGt8r4pFtd0z4mmqwVgR/1kRT321fUWVxC7194MMCkpgEMLQ8tGYyEmAIrRaPCnR4jFBeMTLJsnPKNjlFSw2tnMnqEVKkmqnxMmBGpAIEr2hHHqcHnG02PFrdKL8X0KTEmdlraAQH2GfaWDfRIO4krGYpJa2ggKEdhrktscYia7iB1rE2kMYd2y5SMo7jgCsxVok2gDBetjg0I+h37MHWsZth+CEuOeKzQDLAkQVnBbDKQwmr32r51WSsIVRSkns/5Hf5M32Sx/S+R+daLNELQlL/g6Obamv7fbAcoD19eEnIXwpNEaHIuspnLZH/JVWdnWHOsnHxbSSjQMrnCZ3ePfMvDqOzRZn3odx/KA3cvjvqTVHZABg7z1wRGnN6+gnVgiXwKIUYNX4AM8omle56/0ZiKj0hM=@github.com/viperHTML/viperhtml.github.io.git/': Could not resolve host: EOha0C3HdkKFv97pvBX8QZ7tUXu9QeTamoPuocpHjVTOZxychftqr3EsW9PaSIDjaFSPyKUUywCsf1w53q5NGP11EZpCvEq5bjISDAy5mQnPtB0+J7orVy8lQH5npsYWXSBMqOL6uBOiTWTsgWRVDyni6REeCy8ECEwqBsDIEfZp

travis_time:end:006a4feb:start=1536054507034959270,finish=1536054507054277857,duration=19318587
[0K
[31;1mThe command "./bin/build ../gh-pages" exited with 128.[0m

Done. Your build exited with 1.
```

**Comment by WebReflection** [comment]
OK, it worked now

**Comment by WebReflection** [comment]
So ... `travis-push-update` was pointing at `../gh-pages` but there's not such branch in here so now it's pointing at `../master` ... is this correct?

TL;DR Travis has access but it's not updating anything when I change the layout

**Comment by ExE-Boss** [comment]
It seems like you encrypted the `GH_TOKEN` wrongly:
https://travis-ci.com/viperHTML/viperhtml.github.io/jobs/143597439#L419

**Comment by ExE-Boss** [comment]
You need to add `secure:` between the `-` and first `'` on this line:
https://github.com/viperHTML/viperhtml.github.io/blob/1125d876c02f77afdc0e9f15180dff01cbd30bb5/.travis.yml#L4

**Comment by ExE-Boss** [comment]
Verified fixed: https://github.com/viperHTML/viperhtml.github.io/commit/97ec8e479544c65ff365083aa9b9aa87ec4d5281
